private-templates-service/client/src/components/Common/ShareModal/styled.tsx

```diff
@@ -93,3 +93,11 @@ export const ButtonGroup = styled.div`
 export const CancelButton = styled(DefaultButton)`
   margin-right: 12px;
 `;
+
+export const SendMailButton = styled(DefaultButton)`
+  max-width: fit-content;
```

Comment:
please add this to all buttons in this file:
```
  &: focus {
    outline: 0px;
  }
```

gets rid of the ugly square when u click on a button.

Reply:
addressed